packages/cli/src/ui/commands/chatCommand.ts

```diff
@@ -270,9 +271,126 @@ const deleteCommand: SlashCommand = {
   },
 };
 
+const listAutoCommand: SlashCommand = {
+  name: 'list-auto',
+  description: 'List automatically saved sessions',
+  kind: CommandKind.BUILT_IN,
+  action: async (): Promise<MessageActionReturn> => {
+    const sessions = await listSessions();
+    if (sessions.length === 0) {
+      return {
+        type: 'message',
+        messageType: 'info',
+        content: 'No automatically saved sessions found.',
+      };
+    }
+
+    let message = 'Automatically saved sessions:\n\n';
+    for (const session of sessions) {
+      message += `  ${session.shortId} - ${session.fullId} - ${session.timestamp}\n`;
+    }
+    
+    return {
+      type: 'message',
+      messageType: 'info',
+      content: message,
+    };
+  },
+};
+
+const resumeAutoCommand: SlashCommand = {
+  name: 'resume-auto',
+  description: 'Resume an automatically saved session. Usage: /chat resume-auto <number-or-session-id>',
+  kind: CommandKind.BUILT_IN,
+  action: async (context, args): Promise<SlashCommandActionReturn> => {
+    const input = args.trim();
+    if (!input) {
+      return {
+        type: 'message',
+        messageType: 'error',
+        content: 'Missing session ID. Usage: /chat resume-auto <session-id-or-number>',
+      };
+    }
+
+    // Check if input is a number (short ID)
+    const shortId = parseInt(input, 10);
+    let sessionId = input;
+    
+    if (!isNaN(shortId) && shortId > 0) {
```

Comment:
This is the third time you add this code, can you move the parsing and selection logic (e.g. some new `findSession(input)` to session.ts?